census/vanilla-tf/trainer/model.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Implements the vanilla tensorflow model on single node."""
```

Comment:
I see model.py contain the inference function to build the inference sub-graph. Similarly, it would be good to see functions to build the other sub-graphs here: input (parsing of csv, and transformation logic), training (loss, optimizer), output (the predictions) and evaluation.